extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaInfoboxInsertDialog.scss

```diff
@@ -1,6 +1,42 @@
 .ve-ui-wikiaInfoboxInsertDialog {
 
 	.oo-ui-window-body {
-		overflow: auto
+		overflow: auto;
+	}
+}
+
+.oo-ui-selectWidget {
+	margin-bottom: 5em;
+}
+
+.ve-ui-wikiaInfoboxInsertDialog-add-container {
+	background-color: $color-page;
+	border-top: 1px solid $color-page-border;
```

Comment:
please  consider moving all magic numbers specially the ones that are duplicated into local variables?